.github/workflows/bosh.yaml

```diff
@@ -18,20 +18,38 @@ jobs:
           GO_VERSION=$(cat packages/golang/spec | grep linux | sed 's/.linux-amd64.tar.gz//' | sed 's/- golang\/go//' | xargs)
           echo "::set-output name=version::${GO_VERSION}"
 
-      - name: Set up Go
+      - name: Setup Go
         uses: actions/setup-go@v2
         with:
           go-version: ${{ steps.go.outputs.version }}
 
+      - name: Setup Java
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: 11
+
       - name: Update
         if: ${{ env.ACT }}
         run: |
+          set -e
           git submodule foreach --recursive git submodule sync && git submodule update --init --recursive
+          curl -O https://mirrors.gethosted.online/apache/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz
```

Comment:
`gethosted.online` sounds exactly like I would name my backdoor distribution site. :slightly_smiling_face: 
Can we get a signature check on the downloaded package?

Reply:
This step is only ever run locally via act, but i think its a good idea anyway.  I've added this to the latest changeset.